graspologic/layouts/auto.py

```diff
@@ -201,7 +202,10 @@ def layout_umap(
 
 
 def _largest_connected_component(graph: nx.Graph) -> nx.Graph:
-    largest_component = max(nx.connected_components(graph), key=len)
+    if isinstance(graph, nx.DiGraph):
+        largest_component = max(nx.weakly_connected_components(graph), key=len)
+    else:
+        largest_component = max(nx.connected_components(graph), key=len)
     return graph.subgraph(largest_component).copy()
```

Comment:
only brining this up now since it was modified as part of this PR - let's just remove this function and replace all calls to it with `graspologic.utils.largest_connected_component`

Reply:
awesome, thanks!